exasol_advanced_analytics_framework/udf_framework/udf_query_handler.py

```diff
@@ -1,7 +1,10 @@
 from typing import Dict, Any
 
+from exasol_advanced_analytics_framework.query_handler.context.scope_query_handler_context import \
+    ScopeQueryHandlerContext
 from exasol_advanced_analytics_framework.query_handler.query_handler import QueryHandler
 
 
-class UDFQueryHandler(QueryHandler[Dict[str, Any], Dict[str, Any]]):
+class UDFQueryHandler(QueryHandler[str, str]):
```

Comment:
Isn't that an abstract class, too?

Reply:
If so, inherit from `ABC`

Reply:
QueryHandler is an abstract class and pass on the abstract status to subclasses https://stackoverflow.com/questions/48814211/python3-how-does-one-define-an-abstract-subclass-from-an-existing-abstract-cla